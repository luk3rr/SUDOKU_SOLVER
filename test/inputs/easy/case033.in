040070000 000000200 060490508 007000082 210000609 000046010 000730000 801009300 050010900 
