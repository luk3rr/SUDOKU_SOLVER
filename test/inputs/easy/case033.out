Solving the following grid:
0 4 0 | 0 7 0 | 0 0 0 
0 0 0 | 0 0 0 | 2 0 0 
0 6 0 | 4 9 0 | 5 0 8 
------+-------+------
0 0 7 | 0 0 0 | 0 8 2 
2 1 0 | 0 0 0 | 6 0 9 
0 0 0 | 0 4 6 | 0 1 0 
------+-------+------
0 0 0 | 7 3 0 | 0 0 0 
8 0 1 | 0 0 9 | 3 0 0 
0 5 0 | 0 1 0 | 9 0 0 


Solution found :')

9 4 2 | 5 7 8 | 1 3 6 
7 8 5 | 1 6 3 | 2 9 4 
1 6 3 | 4 9 2 | 5 7 8 
------+-------+------
6 3 7 | 9 5 1 | 4 8 2 
2 1 4 | 3 8 7 | 6 5 9 
5 9 8 | 2 4 6 | 7 1 3 
------+-------+------
4 2 9 | 7 3 5 | 8 6 1 
8 7 1 | 6 2 9 | 3 4 5 
3 5 6 | 8 1 4 | 9 2 7 

Algorithm: BFS
Total time: 118 ms
Total expanded states: 19979
